pkg/apis/core/types.go

```diff
@@ -392,7 +392,6 @@ type PersistentVolumeStatus struct {
 	Reason string
 	// LastPhaseTransitionTime is the time the phase transitioned from one to another
 	// and automatically resets to current time everytime a volume phase transitions.
-	// This is an alpha field and requires enabling PersistentVolumeLastPhaseTransitionTime feature.
```

Comment:
change to `This is a beta field...`

and update the godoc on the feature gate to record the release where it graduated:

```
	// owner: @RomanBednar
	// kep: https://kep.k8s.io/3762
	// alpha: v1.28
	//
	// Adds a new field to persistent volumes which holds a timestamp of when the volume last transitioned its phase.
	PersistentVolumeLastPhaseTransitionTime featuregate.Feature = "PersistentVolumeLastPhaseTransitionTime"
```

Reply:
Done.